src/API/useInAppUpload.js

```diff
@@ -2,17 +2,31 @@ import { useMutation } from 'react-query';
 
 import { useOkapiKy } from '@folio/stripes/core';
 
+import { CAPABILITIES_VALUE } from '../constants';
+
 export const useInAppUpload = () => {
   const ky = useOkapiKy();
 
-  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates }) => {
-    return ky.post(`bulk-edit/${jobId}/item-content-update/upload`, {
-      searchParams: { limit: 10 },
-      json: {
-        itemContentUpdates: contentUpdates,
-        totalRecords: contentUpdates.length,
-      },
-    }).json();
+  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates, capability }) => {
+      const typeOfBulk = CAPABILITIES_VALUE[capability];
+      const getBody = () => {
+            if (typeOfBulk === CAPABILITIES_VALUE.ITEMS) {
+                return {
+                    itemContentUpdate: contentUpdates,
+                    totalRecords: contentUpdates.length,
+                };
+            } else {
+                return {
+                userContentUpdate: contentUpdates,
+                totalRecords: contentUpdates.length,
+                };
+            }
+      };
+
+        return ky.post(`bulk-edit/${jobId}/${typeOfBulk}-content-update/upload`, {
+          searchParams: { limit: 10 },
```

Comment:
why do you need this params for post?

Reply:
@NikitaSedyx can specify which of them?

Reply:
searchParams

Reply:
you do post and define limit 10, any reason?

Reply:
Okay, That logic was written by another guy, so I need to check it out.

Reply:
@NikitaSedyx we need this cause backend requires it and this POST. They should return a limited number of records from request (Looks like GET)

Reply:
interesting :)